CHANGES/4250.misc

```diff
@@ -0,0 +1 @@
+Add two more types to LooseCookies with tests for the allowed types.
```

Comment:
Information about adding tests is not useful for users of `aiohttp`. They are interested in functional differences.

I would not add a new entry, but change `CHANGES/4250.misc`:
```
Fixed annotations of the `cookies` parameter of `CookieJar.update_cookies()` and `ClientRequest.update_cookies()`.
```
(Not sure about format. Should it be Markdown or reStructuredText?)

Reply:
The format is ReST

Reply:
Changed it to what you wrote. Thanks